pep/translations/en-us.json

```diff
@@ -243,6 +243,9 @@
         }
     },
     "relatedInfo": {
+        "title": "Related Info",
+        "batchClose": "Close All",
+        "batchOpen": "Show All",
```

Comment:
these labels should probably use sentence casing to follow the formatting of action/button labels elsewhere in the app